server/database/queries/student/getSpecificAlumni.js

```diff
@@ -0,0 +1,6 @@
+const connection = require('../../config/connection');
+
+const getSpecificAlumni = (id) =>
+  connection.query('select * from student where cohort_id = $1', [id]);
```

Comment:
it would be good if you used uppercase with SQL reserved keywords

Reply:
yes, I forget that :sweat_smile: